packages/data/src/settings/resolvers.js

```diff
@@ -3,20 +3,13 @@
 /**
  * Internal dependencies
  */
-import { NAMESPACE } from '../constants';
 import { updateSettingsForGroup, updateErrorForGroup } from './actions';
+import { getAllSettings } from '@woocommerce/wc-admin-settings';
```

Comment:
I think I mentioned this elsewhere, but this tightly couples the data store to the server provided values and that isn't a good idea.  You should not be hydrating directly in the stores but hydrate via app initialization.  Removing this coupling also removes the need for the resolver.  

So I'd personally restore the HYDRATE_STORE action and use that in app initialization.